docs/source/changelog/0.6.0-changelog.rst

```diff
@@ -0,0 +1,439 @@
+   Made `FadeTransform`, `FadeTransformPieces`, `TransformMatchingShapes` and`TransformMatchingTex` compatible with OpenGL
+
+* `#1343 <https://github.com/ManimCommunity/manim/pull/1343>`__: Make TexTemplate() simple, but keep Tex()'s default template
+   TexTemplate() now returns a simple tex template.
+
+* `#1321 <https://github.com/ManimCommunity/manim/pull/1321>`__: Add OpenGL support to `~.AnimationGroup`
+   Make `~.AnimationGroup` compatible with OpenGL.
+
+* `#1302 <https://github.com/ManimCommunity/manim/pull/1302>`__: Raise appropriate errors in :meth:`~.VMobject.point_from_proportion`
+   - Raise an error if the ``alpha`` argument is not between 0 and 1.
+   - Raise an error if the :class:`~.VMobject` has no points.
+
+* `#1315 <https://github.com/ManimCommunity/manim/pull/1315>`__: Fix performance issues with :meth:`~.VMobject.get_arc_length`, stemming from :pr:`1274`
+
+
+* `#1320 <https://github.com/ManimCommunity/manim/pull/1320>`__: Add `jpeg` extension to the default image extensions
+
+
+* `#1234 <https://github.com/ManimCommunity/manim/pull/1234>`__: New :class:`~.Mobject` method, :meth:`~.Mobject.get_midpoint`
+   Easily get the middle point of a line of an :class:`~.Mobject` with :meth:`~.Mobject.get_midpoint`. (still needs to be adjusted)
+
+* `#1237 <https://github.com/ManimCommunity/manim/pull/1237>`__: Notify user if they are using an outdated version of Manim
+
+
+* `#1308 <https://github.com/ManimCommunity/manim/pull/1308>`__: Improved ManimBanner animations
+
+
+* `#1275 <https://github.com/ManimCommunity/manim/pull/1275>`__: Add SVG <line> element support to :class:`~.SVGMobject`
+
+
+* `#1238 <https://github.com/ManimCommunity/manim/pull/1238>`__: Add parameter `about_point` for `rotate`
+   `about_point` parameter for `rotate`
+
+* `#1260 <https://github.com/ManimCommunity/manim/pull/1260>`__: Change Brace from Tex to SVG (#1258)
+
+
+* `#1122 <https://github.com/ManimCommunity/manim/pull/1122>`__: Support for specifying the interpolation algorithms for individual ImageMobjects
+
+
+* `#1283 <https://github.com/ManimCommunity/manim/pull/1283>`__: Set default value of keyword ``random_seed`` in Scene to None (was 0 and fixed before)
+
+
+* `#1220 <https://github.com/ManimCommunity/manim/pull/1220>`__: Added sanity checks to :meth:`~.Mobject.add_to_back` for Mobjects
+   Add Mobject `add_to_back` sanity checks:
+   - Raises ValueError when Mobject tries to add itself
+   - Raises TypeError when a non-Mobject is added
+   - Filters out incoming duplicate submobjects if at least one instance of that submobject exists in the list
+
+* `#1249 <https://github.com/ManimCommunity/manim/pull/1249>`__: Set corners of :class:`~.Rectangle` in counterclockwise direction
+   Set the corners of `Rectangle` in counterclockwise direction to be consistent with other geometry `Mobject`s. `Transform`ations between `Rectangle` and other geometry `Mobject`s look better now.
```

Comment:
```suggestion
* `#1249 <https://github.com/ManimCommunity/manim/pull/1249>`__: Set corners of :class:`~.Rectangle` in counterclockwise direction to be consistent with other geometry `Mobject`s. `Transform`ations between `Rectangle` and other geometry `Mobject`s look better now.
```

Reply:
It sound a bit repetitive.

Reply:
I modified the body of this: https://github.com/ManimCommunity/manim/pull/1249
@abhi-6988 @abhi3019  Please review and modify if you find the language undescriptive.

Reply:
> I modified the body of this: #1249

Looks good, no issues with that thanks! I've also updated the title and changelog of #1385, please have a look